packages/lit-dev-tools-cjs/src/playground-plugin/renderer.ts

```diff
@@ -168,7 +185,9 @@ class RendererServer {
       process.stdout.write = (() => {}) as any;
       const wds = await startDevServer({
         config: {
-          rootDir: pathlib.resolve(__dirname, '..', '..'),
+          // playground-elements is hoisted to the root node_modules by
```

Comment:
I think ideally we should be doing a node resolve here in case the location changes again.

Reply:
Can't do it here actually, because we're specifying the root directory to serve from. Then we hard code a path into the node_modules folder. Maybe could be done smarter, but requires more refactoring.